app/views/practices/introduction_forms/_image_editor.html.erb

```diff
@@ -13,7 +13,13 @@
     <% end %>
   <div class="dm-cropper-boundary grid-col-12 margin-top-2" data-model-type="<%= practice.class.name.downcase %>">
     <label class="dm-cropper-upload-image-label line-height-26" for="practice_main_display_image">
-      Choose a photograph to represent the Innovation in Search Results.
+      <%=
+        if practice.is_a?(Practice)
+          "Choose a photograph to represent the Innovation in Search Results."
+        else
+          "Choose a photograph to represent the Product"
+        end
+      %>
```

Comment:
@camillevilla what do you think of this change for the `main_display_image` input label text as per Michael's feedback? I figured since the products are not yet included in search results this would make sense for now

Reply:
👎 I would keep the text as is so we don't have to remember to change this field during Search Results update.